docs/benchmarking/adding_benchmarks.md

```diff
@@ -106,11 +106,32 @@ One can enable tracing by following these steps (example is for Golang):
    ```
 3. If the function is a client, use the instrumented grpc dial method to connect to the server:
    ```go
-   conn, err := tracing.DialGRPCWithUnaryInterceptor(addr, grpc.WithInsecure())
+   conn, err := tracing.DialGRPCWithUnaryInterceptor(addr, grpc.WithBlock(), grpc.WithInsecure())
    ```
-
-The producer in the vHive 
-[serving example](/function-images/tests/chained-function-serving/producer/producer.go) 
+4. To enable tracing instrumentation, set `ENABLE_TRACING` environment variable to
```

Comment:
add a note that tracing comes with an overhead so end-to-end measurements should be performed with tracing turned off.

Reply:
also do mention the fact that users need to propagate vhivemetadata throughout their proto files